concordia/consumers.py

```diff
@@ -10,7 +12,11 @@ class AssetConsumer(AsyncJsonWebsocketConsumer):
         await self.channel_layer.group_discard("asset_updates", self.channel_name)
 
     async def asset_update(self, message):
-        await self.send_json({"message": message})
+        await self.send_json(
+            {"message": message, "message_timestamp": int(time.time())}
```

Comment:
`message_timestamp` feels a little long to me. Maybe this should just be something like `sent` which also focuses on the part about this being the transmission time?

Reply:
> It looks like it's working just fine but the filesystem was modified outside of the app. Do we know how that happened?

It doesn't work inside a docker container. I was able to reproduce the problem locally.

Reply:
Done